docs/properties.md

```diff
@@ -54,6 +54,9 @@ used for bridging: every time an envelope is forwarded (no matter if originally
 - An Envelope retry should fail if there are no adapters registered for the envelope's destination chain.
 - A Transaction can only be retried if it has been previously forwarded.
 - The Adapters where the retried Transaction will be sent, should be valid.
+- Forwarding a message should use the number of adapters specified by the configured required confirmations for destination chain
+  - If requiredConfirmations are 0, all adapters should be used.
```

Comment:
This doesn't look so reasonable. We need to get rid of `0` behaviours like the ones on the Aave protocol: `0` means no confirmation required, so I would say it simply should not be a valid configuration (revert on setter)